packages/suite/src/views/suite/SwitchDevice/DeviceItem/WalletInstance.tsx

```diff
@@ -55,11 +56,9 @@ export const WalletInstance = ({
     const { defaultAccountLabelString } = useWalletLabeling();
 
     const deviceAccounts = getAllAccounts(instance.state, accounts);
-    const instanceBalance = getTotalFiatBalance({
-        deviceAccounts,
-        localCurrency,
-        rates: currentFiatRates,
-    });
+
+    const instanceBalance = useTotalFiatBalance(deviceAccounts, localCurrency, currentFiatRates);
```

Comment:
I would call it `walletBalance` in both files

Reply:
done